src/Tribe/RSVP.php

```diff
@@ -922,139 +890,113 @@ public function create_attendee_for_ticket( $ticket, $attendee_data ) {
 			update_post_meta( $product_id, '_stock', $stock );
 		}
 
-		/**
-		 * RSVP specific action fired when a RSVP-driven attendee ticket for an event is generated.
-		 * Used to assign a unique ID to the attendee.
-		 *
-		 * @param int    $attendee_id ID of attendee ticket.
-		 * @param int    $post_id     ID of event.
-		 * @param string $order_id    RSVP order ID (hash).
-		 * @param int    $product_id  RSVP product ID.
-		 */
-		do_action( 'event_tickets_rsvp_attendee_created', $attendee_id, $post_id, $order_id, $product_id );
-
-		/**
-		 * Action fired when an RSVP attendee ticket is created.
-		 * Used to store attendee meta.
-		 *
-		 * @param int $attendee_id       ID of the attendee post.
-		 * @param int $post_id           Event post ID.
-		 * @param int $product_id        RSVP ticket post ID.
-		 * @param int $order_attendee_id Attendee # for order.
-		 */
-		do_action( 'event_tickets_rsvp_ticket_created', $attendee_id, $post_id, $product_id, $order_attendee_id );
-
-		if ( null === $order_attendee_id ) {
-			/**
-			 * Action fired when an RSVP ticket has had attendee tickets generated for it.
-			 *
-			 * @param int    $product_id RSVP ticket post ID.
-			 * @param string $order_id   ID (hash) of the RSVP order.
-			 * @param int    $qty        Quantity ordered.
-			 */
-			do_action( 'event_tickets_rsvp_tickets_generated_for_product', $product_id, $order_id, 1 );
-
-			$this->clear_attendees_cache( $post_id );
-		}
-
 		return $attendee_id;
 	}
 
 	/**
 	 * Update the RSVP values for this user.
 	 *
-	 * Note that, within this method, $order_id refers to the attendee or ticket ID
+	 * Note that, within this method, $attendee_id refers to the attendee or ticket ID
 	 * (it does not refer to an "order" in the sense of a transaction that may include
 	 * multiple tickets, as is the case in some other methods for instance).
 	 *
-	 * @param array $data
-	 * @param int   $order_id
-	 * @param int   $event_id
+	 * @param array $attendee_data Information that we are trying to save.
+	 * @param int   $attendee_id   The attendee ID.
+	 * @param int   $post_id       The event/post ID.
 	 */
-	public function update_attendee_data( $data, $order_id, $event_id ) {
+	public function update_attendee_data( $attendee_data, $attendee_id, $post_id ) {
+		// Bail if the user is not logged in.
+		if ( ! is_user_logged_in() ) {
+			return;
+		}
```

Comment:
Cap check?

Reply:
Further below in the logic is makes sure the attendee matches one 'owned' by the current user ID.